webclient/src/components/views/ProjectList.vue

```diff
@@ -14,31 +14,39 @@
         <Project v-for="project in projects" :project="project" :key="project.ID" v-on:update="Update" />
       </div>
     </div>
+    <div v-if="!loading">
+      <vue-simple-spinner></vue-simple-spinner>
+    </div>
   </div>
 </template>
 
 <script>
 import { GetProjects } from '@/database';
 import Project from '@/components/elements/Project';
 import LoggedInHeader from '@/components/elements/LoggedInHeader';
+import VueSimpleSpinner from 'vue-simple-spinner/src/components/Spinner';
 
 export default {
   components: {
+    VueSimpleSpinner,
```

Comment:
Is it possible to keep this like the rest of the components for consistency?

`'vue-simple-spinner': VueSimpleSpinner`